.github/workflows/tests-arm64-nightly.yaml

```diff
@@ -1,18 +0,0 @@
----
-name: Integration Arm64 Nightly
-permissions: read-all
-on:
-  # schedules always run against the main branch, hence we have to create separate jobs
-  # with individual checkout actions for each of the active release branches
-  schedule:
-    - cron: '30 2 * * *' # runs daily at 2:30 am.
-jobs:
-  main-arm64:
-    uses: ./.github/workflows/tests-arm64-template.yaml
-    with:
-      etcdBranch: main
-  release-35-arm64:
```

Comment:
Are you going to take care of release-3.5 in a separate PR?

Reply:
Yes, I'll create a new PR targeting the release-35 branch.